sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -751,25 +754,25 @@ export class EventHubReceiver extends LinkEntity {
           options = this._createReceiverOptions(receiverOptions);
         }
 
-        log.error(
+        log.info(
```

Comment:
This should be using the `verbose` channel.

We have already logged to the info channel that we are attempting to initializing the link. More details around the options being used would be candidate for `verbose`